### PR TITLE
Rename to zone watcher

### PR DESCRIPTION
**Pull request by jjbuchan**
Renaming from management to watcher.

I changed my mind and also renamed the package.  Since it's not depended on by any other services it's easy to change it here now.

I wonder if devops needs to change anything?  @shawnashlee @terryllowery @dgvigil

### CODE REVIEW

File: src/main/java/com/rackspace/salus/zone_watcher/ZoneManagementApplication.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.rackspace.salus.zone_mgmt;
+package com.rackspace.salus.zone_watcher;
```
Comment (itzg) [inline src/main/java/com/rackspace/salus/zone_watcher/ZoneManagementApplication.java]: I tend not to like underscores in package names...I was originally following the pattern of our other modules. Maybe this is the opportunity to switch it to just `zw`. Textually-short package paths are good too.

Comment (jjbuchan) [reply src/main/java/com/rackspace/salus/zone_watcher/ZoneManagementApplication.java]: I'm not big on `_` either.

**Comment by shawnashlee** [comment]
we would need to change things in both google cloud and spinnaker

**Comment by jjbuchan** [comment]
@shawnashlee I'll create a jira for you guys so it doesn't get left behind

**Pull request merged**